backend/LexBoxApi/LexBoxApi.csproj

```diff
@@ -11,24 +11,25 @@
 
     <ItemGroup>
         <PackageReference Include="DataAnnotatedModelValidations" Version="4.1.0" />
-        <PackageReference Include="HotChocolate.Types.Analyzers" Version="13.5.1">
+        <PackageReference Include="HotChocolate.Analyzers" Version="13.6.0-preview.21" />
+        <PackageReference Include="HotChocolate.Types.Analyzers" Version="13.6.0-preview.21">
             <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
             <PrivateAssets>all</PrivateAssets>
         </PackageReference>
-        <PackageReference Include="HotChocolate.AspNetCore" Version="13.5.1" />
-        <PackageReference Include="HotChocolate.AspNetCore.Authorization" Version="13.5.1" />
-        <PackageReference Include="HotChocolate.Data.EntityFramework" Version="13.5.1" />
-        <PackageReference Include="HotChocolate.Diagnostics" Version="13.5.1" />
+        <PackageReference Include="HotChocolate.AspNetCore" Version="13.6.0-preview.21" />
```

Comment:
HotChocolate.AspNetCore now has 13.6.1 and 13.7.0 versions available, so this (and following lines) could probably be bumped to a release version instead of a preview.